changelog/add-romania-fees

```diff
@@ -0,0 +1,4 @@
+Significance: major
+Type: add
+
+Add currency fees for RON (The currency of Romania)
```

Comment:
IMO, it doesn't reflect the actual scope of changes. There are no currency-related changes in this PR. The title of #6088 is I think more appropriate here.

Reply:
changed. thx.